ProjectCoimbra.UWP/Project.Coimbra/Helpers/SongPagesHelper.cs

```diff
@@ -2,14 +2,14 @@
 
 namespace Coimbra.Helpers
 {
+    using Coimbra.Communication;
```

Comment:
Please keep the sort order in the standard format, with `System` usings going first.

Reply:
yup sorry, autoformatter on my end reorganized those